app/assets/stylesheets/_tabs.scss

```diff
@@ -0,0 +1,112 @@
+.entries-tabs-container{
+  margin-left: -25px;
+  margin-right: -25px;
+}
+
+.reports-tabs-container{
+  @include media(max-width $medium-screen){
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Opening curly brace `{` should be preceded by one space